.storybook/main.js

```diff
@@ -75,7 +76,7 @@ module.exports = {
 			...config,
 			stats: {
 				/* Suppress autoprefixer warnings from storybook build */
-				warningsFilter: [/autoprefixer: /],
+				warningsFilter: [/autoprefixer: /, /postcss-*/, /stylelint/],
```

Comment:
Strips out the postcss & stylelint warnings from the command line output for storybook

Reply:
Does this mean on `yarn start`, we shouldn't see any more stylelint warnings? Just curious as I'm still getting some "custom property not defined" warnings that look like they're from stylelint. Not something that I think should hold up the PR 🙂

Reply:
This means you won't see duplicate messages - you'll still see the pretty-print output from postcss-reporter but you won't see these duplicates that are prefixed with WARN and piped out of webpack: 

<img width="1219" alt="Screenshot 2024-03-21 at 1 32 49 PM" src="https://github.com/adobe/spectrum-css/assets/1840295/98cc0fa7-2689-47ca-b7fc-e1ae00ab40be">